hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/HoodieCopyOnWriteTableInputFormat.java

```diff
@@ -76,6 +78,7 @@
  * NOTE: This class is invariant of the underlying file-format of the files being read
  */
 public class HoodieCopyOnWriteTableInputFormat extends HoodieTableInputFormat {
+  private static final Logger LOG = LogManager.getLogger(HoodieCopyOnWriteTableInputFormat.class);
```

Comment:
Is this still needed?

Reply:
this actually got merged now. https://github.com/apache/hudi/pull/6161, so if i rebase it will basically be the same and is needed in general.